src/main/java/com/redhat/red/build/finder/BuildFinder.java

```diff
@@ -70,27 +69,49 @@
 
     private File outputDirectory;
 
+    private MultiValuedMap<String, Integer> checksumMap;
```

Comment:
This is being used to store checksum -> build ids. It's not hit all that often as it will only be hit on duplicates, but it prevents a Brew lookup, which is expensive.